<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clientWidget</class>
 <widget class="QWidget" name="clientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>clientWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" rowstretch="5,0" columnstretch="0">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="topMargin">
    <number>11</number>
   </property>
   <property name="rightMargin">
    <number>11</number>
   </property>
   <property name="bottomMargin">
    <number>11</number>
   </property>
   <property name="horizontalSpacing">
    <number>7</number>
   </property>
   <item row="0" column="0" rowspan="2">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="_2">
      <property name="leftMargin">
       <number>11</number>
      </property>
      <property name="rightMargin">
       <number>11</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEditPort">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Port:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEditIp">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>IP:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditNickName">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>nickname:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxOnline">
        <property name="title">
         <string>在线列表：</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
